src/property.h

```diff
@@ -135,6 +136,12 @@ struct Property : public detail::Property {
     #if defined(THZ_5_5_ECO)
         PROPERTY(SOMMERBETRIEB_HYSTERESE, 0x05a2, Type::et_dec_val);
     #endif
+    PROPERTY(VERFLUESSIGERTEMPERATUR, 0x059c, Type::et_dec_val);
+    PROPERTY(VERDICHTER_HEIZEN, 0x05a4); 
+    PROPERTY(VERDICHTER_KUEHLEN, 0x05a5); 
+    PROPERTY(VERDICHTER_WW, 0x05a6); 
+    PROPERTY(EL_NE_HEIZEN, 0x05a7); 
+    PROPERTY(EL_NE_WW, 0x05a8); 
```

Comment:
Why do you need these?
If you want to know which operating mode the WP is in, there are already sensors for that:
```
    name: "VERDICHTER"
    id: VERDICHTER

  - platform: template
    name: "HEIZEN"
    id: HEIZEN

  - platform: template
    name: "KUEHLEN"
    id: KUEHLEN

  - platform: template
    name: "WARMWASSERBEREITUNG"
    id: WARMWASSERBEREITUNG

  - platform: template
    name: "ELEKTRISCHE_NACHERWAERMUNG"
    id: ELEKTRISCHE_NACHERWAERMUNG
```

Reply:
The values I wanted to add do not show the operating mode of the WP, it shows the runtime in hours. 

I think a fundamental decision needs to be made. Do you want to only include and display the values that interest you, or do you want to enable everything that the heat pump offers? Personally, I prefer the second approach and have solved it by initially including everything in the code but setting most values in HA to ‘disabled_by_default’. This way, I can select and display the values that interest me in HA. Mainly, these are statistical values.

Reply:
Ok, but then please set the update_interval to 1h